internal/codeintel/policies/internal/store/configurations_test.go

```diff
@@ -71,73 +82,84 @@ func TestGetConfigurationPolicies(t *testing.T) {
 	}
 
 	var (
-		vt = true
-		vf = false
+		True  = true
+		False = false
```

Comment:
```suggestion
		true_  = true
		false_ = false
```

(Or `trueValue`, `falseValue`)

Minor suggestion: Use lowercase local variable names similar to the other Go code, it reads weirdly otherwise.